indico/htdocs/sass/modules/_category.scss

```diff
@@ -77,6 +76,39 @@
     margin: 0 0 2em 0;
 }
 
+
+.category-statistics {
+    max-width: 1600px;
+    .i-box {
+        margin-left: 0.8em;
+        margin-right: 0.8em !important;
+        width: 45%;
+        &:first-child {
+            margin-left: 0;
+        }
+        .i-box-header {
+            text-align: left;
+            .i-box-title {
+            }
+        }
+        .i-box-content {
+            display: inline-block;
+
+            width: 96%;
+            height: 350px;
+            padding: 2% 2% 0 2%;
+        }
+    }
+    .i-badges-list > .i-badge {
+        width: 45%;
+        &:first-child {
+            margin-left: 0;
+        }
+    }
+}
+
+
+
 .category-sidebar {
     @include box-sizing(border-box);
     background: #FAFAFA;
```

Comment:
I had a look but we don't have those colors in the palette and it is used everywhere in category-related styles so it would be a big change in the legacy code to fix this.